unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/PoolManager/Pool.cs

```diff
@@ -113,6 +113,18 @@ private PoolableObject Extract()
             return po;
         }
 
+        public async UniTask IterativePrewarm(int prewarmCount)
```

Comment:
I would call it `AsyncPrewarm`. We need a cancellation token here to dispose this task properly if the caller is disposed as well.

It can also mess up with a normal prewarming right? how is that edge case handled?